content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -368,15 +367,12 @@ to `Retain`, including cases where you are reusing an existing PV.
 Support for expanding PersistentVolumeClaims (PVCs) is enabled by default. You can expand
 the following types of volumes:
 
-* azureDisk
-* azureFile
-* awsElasticBlockStore
-* cinder (deprecated)
+* azureFile (deprecated)
 * {{< glossary_tooltip text="csi" term_id="csi" >}}
 * flexVolume (deprecated)
-* gcePersistentDisk
+* gcePersistentDisk (deprecated)
 * rbd
-* portworxVolume
+* portworxVolume (deprecated)
```

Comment:
Please remove `(deprecated)` to avoid confusion. Portworx in-tree driver is not deprecated as our CSI Migration is not enabled by default.

Reply:
Hi @ggriffiths, the in-tree driver is marked deprecated as soon as CSI migration moves to Beta off-by-default.  "deprecated" means it is still supported but we want users to move to CSI driver ASAP.  So this should be marked as "deprecated".  We usually have a message about "deprecation" in the release note as well when CSI migration moves to beta.

To differentiate this from a deprecated driver that does not go through CSI migration, may be we can add note here to indicate that.

Reply:
AIUI we can deprecate an in-tree driver so long as there is an alternative we document (even if that is “use a different kind of storage”). We don't need to wait for the automated migration.

If Kubernetes has deprecated `portworxVolume` in-tree, that's what we should document.